common/src/main/java/cz/incad/kramerius/utils/RESTHelper.java

```diff
@@ -41,13 +41,6 @@ public static InputStream inputStream(String urlString, String user, String pass
             if (httpUrl != null) {
                 int responseCode = httpUrl.getResponseCode();
                 LOGGER.severe("status code "+responseCode);
-                
```

Comment:
Proč je smazaný tento blok kódu? Ten resthelper se používá na mnoha místech, aby to něco nerozbilo....

Reply:
Aha, jasně, díky za vysvětlení. Doplnil bys ale prosím do toho loggeru url? Ta může být užitečná.

Např. "http://fedora/blabla returned status code 404"

Reply:
Ano, hotovo.